ps/module.mk depends on vars set up in proc/module.mk, so make sure we always
include the .mk files in the right order

--- procps-3.2.8/Makefile
+++ procps-3.2.8/Makefile
@@ -174,7 +174,8 @@
 # want this rule first, use := on ALL, and ALL not filled in yet
 all: do_all
 
--include */module.mk
+-include proc/module.mk
+-include ps/module.mk
 
 do_all:    $(ALL)
 
